sale_stock_available_to_promise_release/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    display_is_available = fields.Boolean(
```

Comment:
IMO This should be true by default and related to the company. You maybe want to show this field only in some companies